src/config.rs

```diff
@@ -228,10 +228,26 @@ pub(crate) struct NotifyZulipConfig {
 pub(crate) struct NotifyZulipLabelConfig {
     pub(crate) zulip_stream: u64,
     pub(crate) topic: String,
-    pub(crate) message_on_add: Option<String>,
-    pub(crate) message_on_remove: Option<String>,
-    pub(crate) message_on_close: Option<String>,
-    pub(crate) message_on_reopen: Option<String>,
+    #[serde(rename = "message_on_add", default, deserialize_with = "string_or_seq")]
+    pub(crate) messages_on_add: Vec<String>,
+    #[serde(
+        rename = "message_on_remove",
+        default,
+        deserialize_with = "string_or_seq"
+    )]
+    pub(crate) messages_on_remove: Vec<String>,
+    #[serde(
+        rename = "message_on_close",
+        default,
+        deserialize_with = "string_or_seq"
+    )]
+    pub(crate) messages_on_close: Vec<String>,
+    #[serde(
+        rename = "message_on_reopen",
```

Comment:
Should all the renames be alias? I guess it doesn't matter for just Deserialize?